pandas/tests/extension/test_arrow.py

```diff
@@ -2212,3 +2212,11 @@ def test_mode(data_for_grouping, dropna, take_idx, exp_idx, request):
     result = ser.mode(dropna=dropna)
     expected = pd.Series(data_for_grouping.take(exp_idx))
     tm.assert_series_equal(result, expected)
+
+
+def test_global_int_lists():
```

Comment:
IMO, I don't think we need to test `SIGNED_INT_PYARROW_DTYPES` and `UNSIGNED_INT_PYARROW_DTYPES` have the right items. We don't really test the constants in `pandas/_testing/__init__.py` and it was more of a typo. (Appreciate the gesture though!)

Reply:
Removed: https://github.com/pandas-dev/pandas/pull/48457/commits/2f4c7bcf497b89227e109a3dff7fb96bfd903948